src/ray/common/ray_config_def.h

```diff
@@ -189,6 +189,17 @@ RAY_CONFIG(int64_t, worker_cap_max_backoff_delay_ms, 1000 * 10)
 /// even balancing of load. Low values (min 0.0) encourage more load spreading.
 RAY_CONFIG(float, scheduler_spread_threshold, 0.5)
 
+/// Used by the default hybrid policy only. The scheduler will randomly pick
+/// one node from the top k in the cluster to improve load balancing. The
+/// scheduler guarantees k is at least equal to this fraction * the number of
+/// nodes in the cluster.
+RAY_CONFIG(float, scheduler_top_k_fraction, 0.1);
+
+/// Used by the default hybrid policy only. The scheduler will randomly pick
+/// one node from the top k in the cluster to improve load balancing. Set this
+/// value to a positive integer to set k.
+RAY_CONFIG(int32_t, scheduler_top_k_absolute, -1);
```

Comment:
Mention this is just for testing? Also this should precedent the scheduler_top_k_fraction? can you mention that from the doc?